src/encoded/static/components/search.js

```diff
@@ -435,12 +444,27 @@ const ExperimentComponent = (props, reactContext) => {
             <div className="result-item">
                 <div className="result-item__data">
                     <a href={result['@id']} className="result-item__link">
+                        {(isFunctionalExperiment && result.perturbation_type) ?
```

Comment:
This code is fine. Still, it is best not to have Ternary operator nested in a Ternary operator. There is no rule against it, it just gets a little harder to follow in my personal opinion.